### ISSUE TITLE
Clarity improvements for wiki page

### ISSUE THREAD

**Issue by matthewlee01** [processing/p5.js#6506]
### Increasing Access

Hi, I was recently getting started with learning p5.js and noticed 2 small clarity issues on this wiki page: https://github.com/processing/p5.js/wiki/Local-server

1. The "Node http-server" heading is incorrectly formatted as a subheading.
2. The instruction directing readers to the "lightning bolt" icon should be annotated with an emoticon (⚡️) to match the formatting of the rest of the page, as well as improving clarity for readers using a page translator.

### Most appropriate sub-area of p5.js?

- [X] Accessibility
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Build Process
- [ ] Unit Testing
- [ ] Internalization
- [ ] Friendly Errors
- [ ] Other (specify if possible)

### Feature enhancement details

When skimming through this documentation, I initially missed the node.js option because it was formatted as a subheading; then once I saw it I was confused and interpreted it as a sub step of the previous Python section.

The emoticon/icon is just nice polish to keep the page more cohesive (see the other emoticon usage in the same page) and accessible.

**Comment by nickmcintyre** [comment]
Thanks for pointing these out @matthewlee01! We'll incorporate your suggestions in the next iteration of this page. It'll be updated merged with a few others over the coming months.

**Comment by Garima3110** [comment]
Also, @nickmcintyre  I would like to point out that the link for cloning its repository doesn't work . 

Link : https://github.com/processing/p5.js.wiki.git

Please let me know if a separate issue should  be opened for this?!
Thankyou!!

**Comment by nickmcintyre** [comment]
@Garima3110 give `git clone https://github.com/processing/p5.js` a try. Here are our [contributor guidelines](https://github.com/processing/p5.js/blob/main/contributor_docs/contributor_guidelines.md) with more detailed info for different ways to contribute.

**Comment by Garima3110** [comment]
> @Garima3110 give `git clone https://github.com/processing/p5.js` a try. Here are our [contributor guidelines](https://github.com/processing/p5.js/blob/main/contributor_docs/contributor_guidelines.md) with more detailed info for different ways to contribute.

@nickmcintyre  ,I think you got me wrong, actually the wiki page had a clone link to contribute to that page, I copied the same link and didn't work. 

Screenshot of the same:

<img width="945" alt="image" src="https://github.com/processing/p5.js/assets/110815240/9e7b8c79-232f-4e4c-bb8d-c51f91ff1db6">

So ,I just wanted to know whether an issue should be opened for this or not?!
It would be very helpful if you could respond to this.
Thankyou!

**Comment by nickmcintyre** [comment]
@Garima3110 ohh gotcha. Have you tried `git clone https://github.com/processing/p5.js.wiki.git`? I just tested it on macOS and Linux.

We'll probably archive the wiki soon as part of a larger project to make the p5.js documentation more accessible. Most of the content will find a new home in the repo and/or on the website.

**Comment by Garima3110** [comment]
@nickmcintyre ,Thanks for your response.
Here is the screenshot of trying `git clone https://github.com/processing/p5.js.wiki.git` on windows.

Also, on checking status ,its just showing some deleted files.

<img width="804" alt="image" src="https://github.com/processing/p5.js/assets/110815240/321f1b7b-97a8-42bc-87de-edfec65c9bdd">

**Comment by limzykenneth** [comment]
The wiki is not publicly editable and the relevant changes have been made. There is a rewrite of Wiki contents currently in the works so there's no plan to accept changes proposals for the Wiki for the moment. Thanks.
